internal/goofys_test.go

```diff
@@ -1956,3 +1956,73 @@ func (s *GoofysTest) TestRead403(t *C) {
 	_, err = fh.ReadFile(0, buf)
 	t.Assert(err, Equals, syscall.EACCES)
 }
+
+func (s *GoofysTest) TestRmdirWithDiropen(t *C) {
```

Comment:
this test passes with unmodified goofys

Reply:
I see, that's because in goofys_test the tests are by default with with 0 type cache ttl and stat cache ttl. Once I enable those I can make the test fail